src/libraries/testPackages/build/packageTest.targets

```diff
@@ -72,6 +72,10 @@
     <ItemGroup>
       <ReferenceLibToTest Include="@(ReferencePath)" Condition="'%(ReferencePath.NuGetPackageId)' == '$(TestPackageId)'" />
       <RuntimeLibToTest Include="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.NuGetPackageId)' == '$(TestPackageId)'" />
+      <!-- Don't verify RID specific libs that aren't compatible on .NETFramework, i.e. Unix. -->
+      <RuntimeLibToTest Remove="@(RuntimeLibToTest)" Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' and
+                                                                '%(RuntimeLibToTest.RuntimeIdentifier)' != '' and
+                                                                '%(RuntimeLibToTest.RuntimeIdentifier)' != 'win'" />
```

Comment:
I think you want startswith('win-') here.  Here's where NuGet calculates the RID for classic CSProj:
https://github.com/dotnet/NuGet.BuildTasks/blob/b2f0c162994237c4a9c23af79d95f79847499ebd/src/Microsoft.NuGet.Build.Tasks/Microsoft.NuGet.targets#L89-L113

Reply:
`win-` wouldn't catch "win" itself so I guess checking (eq "win" or startswith("win-")) is necessary:

![image](https://user-images.githubusercontent.com/7412651/132756335-281a59c1-8673-4413-92d3-5290534429e4.png)